scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -2039,3 +2040,123 @@ def test_compare_as_davenport_as_euler():
             eul = rot.as_euler(seq)
             dav = rot.as_davenport(ax, order)
             assert_allclose(eul, dav, rtol=1e-12)
+
+
+def test_zero_length_rotation_construction():
```

Comment:
The name can be `test_zero_length_rotation_construction_and_representation`.

Reply:
I separate the 0-rotation tests, into more separate test function. I gathered assert statements which were similar. So all names are now different.

Let me know if you have suggestions.